src/smallestPrimeFactor.js

```diff
@@ -12,14 +12,27 @@
  * @returns {number} The smallest prime factor of num
  */
 function smallestPrimeFactor(num) {
-  // This is your job. :)
+  primeFactors = [3, 5, 7, 11, 13]
```

Comment:
1. Without `let` this becomes a global variable.
1. You can't enumerate all possible prime factors in a single array (there are infinitely many). How would you find the smallest prime factor of a number using pen and paper? Work that out and then translate to code.